CONTRIBUTORS.txt

```diff
@@ -12,3 +12,4 @@ Content and improvements:
     E99p1ant (@wuhan005)
     Jenaye (@jenaye)
     Maraudery (@maraudery)
+    Neeraj Babu (@SAAneeraj)
```

Comment:
You can also add your contributor badge to https://github.com/noraj/rawsec-cybersecurity-inventory/blob/master/README.md#contributions

Reply:
Sry I did'nt really understand it can you please guide me

Reply:
I figured it out and made another pull request to main branch